server/src/com/mirth/connect/connectors/http/HttpDispatcher.java

```diff
@@ -335,6 +335,7 @@ public Response send(ConnectorProperties connectorProperties, ConnectorMessage c
                 list.add(header.getValue());
             }
 
+            connectorMessage.getConnectorMap().put("responseStatusCode", String.valueOf(statusCode));

```

Comment:
I think my preference would be to leave it as an `int` rather than converting it to a `String`. It is a numerical value, after all.

It won't have the issue with velocity adding the `.0` that @jonbartels alluded to because java auto-boxing will store it as a `java.lang.Integer` rather than a `java.lang.Double` as it would be if you put a JavaScript number into the map.

Reply:
I agree that the statusCode should probably be left as an `int`.

Reply:
Well sure...would want to do response code comparisons by integer ranges anyway.   

200s are OK
300s are redirects. 
400s are errors

Reply:
Updated.